gunicorn/config.py

```diff
@@ -110,7 +117,7 @@ def worker_class_str(self):
         return uri
 
     @property
-    def worker_class(self):
+    def worker_class(self) -> Type[SyncWorker]:
```

Comment:
maybe better to use the Base worker class or create a type for it? this function will indeed return different type of worker class

Reply:
Cool, I change that :-) Should I add it to this commit (and force-push) or would you rather like separate commits?

Reply:
at this stage i think it's always bette to add it as a commit. It will always be time later to fix the history.